### DISCUSSION TITLE
Email Lost (Received on Server but neither delivered, nor Bounced)

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#2273] [Help with using Postal]
Hello team,

i have recieved an email on server as you can in the log smtp_server.log

<img width="601" alt="image" src="https://user-images.githubusercontent.com/32220165/205635508-a4f0ec5c-7ef5-4e62-8612-12ec7d12e2fe.png">

But Neither delivered, nor bounce, since not getting that email in smtp_sender.log 

Not showing on Interface (UI) as well 

approx size of mail was 15 MB

one observation was found that , smiliar mail id was found in slow_message_db_queries.log

<img width="642" alt="image" src="https://user-images.githubusercontent.com/32220165/205636154-145d1be3-3409-4b2e-8f5a-36f729724650.png">

what may be the reason for the same,

Note: all other emails are going well with the server

Please do advise

thank in advance

**Comment by willpower232** [comment]
It is a little hard to see that your message was successfully received from the screenshot but as you say, things are usually working so more likely to be related to the specific size of the message.

You might need to have a look at the other mysql log files to see what happened (or you could rummage deeper in the Postal logs to see any error traces). Some have found that they need to update the MySQL settings to allow it to handle larger piles of information, see https://github.com/postalserver/postal/issues/324#issuecomment-323538575
